bin/cli.js

```diff
@@ -70,6 +70,22 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			// When webpack is not installed and no args passed to the CLI
+			if (err.code === "MODULE_NOT_FOUND") {
+				let errorMessage =
+					"\n\u001b[31mwebpack not found, please install webpack using\n\t\u001b[33mnpm install --save-dev webpack\n";
+
+				if (process.env.npm_execpath !== undefined) {
+					if (process.env.npm_execpath.indexOf("yarn") !== -1) {
```

Comment:
I think includes is the preferred way nowadays e.g.
```suggestion
					if (process.env.npm_execpath.includes("yarn")) {
```